src/Autohide.js

```diff
@@ -182,10 +186,51 @@ define(function (require, exports, module) {
         exposeDelay = delay;
     }
 
+    function showDelayDialog() {
+        var Dialog = Dialogs.showModalDialog(
+            brackets.DIALOG_ID_SAVE_CLOSE,
+            Strings.AUTOHIDE_DELAY_MODAL_TITLE,
+            Mustache.render(sliderTemplate, {
+                exposeDelay: exposeDelay + Strings.AUTOHIDE_DELAY_MODAL_MS_LABEL
+            }),
+            [{
+                className: Dialogs.DIALOG_BTN_CLASS_PRIMARY,
+                id: "outline.cancel",
+                text: Strings.AUTOHIDE_DELAY_MODAL_BUTTON_CANCEL
```

Comment:
In order to reduce the number of translations and stay closer to Bracket's UX, I would go with the standard dialog and only have one Ok-button:
http://brackets.io/docs/current/modules/widgets/Dialogs.html#-showModalDialog

That way, we also get Close-On-Esc for free (which could be seen as cancel).

Reply:
Fixed.